docs/DOCKER.rst

```diff
@@ -47,19 +39,28 @@ For running an S3 AWS backend, you will have to add a new section
             "credentialsProfile": "default"
         }
     }
+(...)
 
-You will also have to mount your AWS credentials file:
+You will also have to edit your AWS credentials file to be able to use your
+command line tool of choice. This file should only state credentials for S3 AWS
+hosted data, not for your Zenko CloudServer hosted data.
```

Comment:
I don't understand this sentence.  You can have numerous profiles in your aws credentials file. they need to mount the file that has any profile mentioned in their aws location.  Above, where it says "credentialsProfile":"default", this could say any other profile they choose.

Reply:
You're right, I wrote this in the hackathon context for a specific team, but that's innacurate in the greater world. Providing an alternative.